x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/popover_editor.tsx

```diff
@@ -291,131 +303,116 @@ export function PopoverEditor(props: PopoverEditorProps) {
               setInvalidOperationType(null);
             }}
           />
-        </EuiFlexItem>
-        <EuiFlexItem>
-          <EuiFlexGroup gutterSize="s">
-            <EuiFlexItem grow={null} className={classNames('lnsIndexPatternDimensionEditor__left')}>
-              <EuiListGroup gutterSize="none" listItems={getSideNavItems()} />
-            </EuiFlexItem>
-            <EuiFlexItem grow={true} className="lnsIndexPatternDimensionEditor__right">
-              {incompatibleSelectedOperationType && selectedColumn && (
-                <>
-                  <EuiCallOut
-                    data-test-subj="indexPattern-invalid-operation"
-                    title={i18n.translate('xpack.lens.indexPattern.invalidOperationLabel', {
-                      defaultMessage: 'To use this function, select a different field.',
-                    })}
-                    color="warning"
-                    size="s"
-                    iconType="sortUp"
-                  />
```

Comment:
This callout is also gone in favor of `helpText` on the input itself. Which means the DTS's for both callouts are gone and I'll need help updating the tests.

<img width="374" alt="Screen Shot 2020-08-26 at 19 07 44 PM" src="https://user-images.githubusercontent.com/549577/91365555-711a0a80-e7cf-11ea-8e4c-40030cef9ac1.png">